cardano-cli/test/cardano-cli-golden/files/golden/help/alonzo_genesis_create-testnet-data.cli

```diff
@@ -7,8 +7,8 @@ Usage: cardano-cli alonzo genesis create-testnet-data [--spec-shelley FILE]
                                                         [--drep-keys INT]
                                                         [--stuffed-utxo INT]
                                                         [--utxo-keys INT]
-                                                        [--supply LOVELACE]
-                                                        [--supply-delegated LOVELACE]
+                                                        [--total-supply LOVELACE]
+                                                        [--delegated-supply LOVELACE]
```

Comment:
```suggestion
                                                        [--supply-total LOVELACE]
                                                        [--supply-delegated LOVELACE]
```
I think `supply` prefix may be more intuitive. I'm not sure - current version is fine too.

Reply:
Actually I agree with you, but I chose to stay consistent with the existing flags, e.g. in `--drep-keys`, `--utxo-keys`, and `--genesis-keys`; the prefix is the disambiguation. By having `--supply-total` and `--supply-delegated`, I stick to this convention :shrug: